wikimedia/wikipedia-ios
# Add article image to diff header view if available

**Phabricator:** https://phabricator.wikimedia.org/T335579#9007269

### Notes
* This PR adds a new DiffHeaderView that wraps the existing DiffHeaderTitleView, to add an image view if necessary, with little edits to the existing .xib file
* If the article has an image, it will be displayed on the header of the diff view

### Test Steps
1. Go to a diff on an article with a lead image
2. Confirm that it shows on the diff view according to the designs
3. Go to an article that has no lead image, then make sure its diff view is shown correctly
